library/kotlin/src/io/envoyproxy/envoymobile/RetryPolicyMapper.kt

```diff
@@ -30,6 +30,7 @@ internal fun RetryPolicy.outboundHeaders(): Map<String, List<String>> {
  *
  * @return The string representation of the retry rule
  */
+ //TODO: change to enum pattern in UpstreamHttpProtocol
```

Comment:
Is this still relevant since you've added the `enumValue()` method in the class?

Reply:
yep I mean I wanted to update the enum classes to be `enum class RetryPolicy(internal val stringValue: String) {...}` to avoid having the `stringValue()` method. I have updated RequestMethod and RetryPolicy.